src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react";
+import { getReviews } from "../utils/api";
+import "../App.css";
+
+const ReviewsList = () => {
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    getReviews().then((reviews) => {
+      setReviews(reviews);
+    });
+  }, []);
+  return (
+    <main>
+      <div>
+        <ul className="reviews-list">
+          {reviews.map((review) => {
```

Comment:
This definitely works although results quite a large Reviewslist component, it would be nice to have the 'review cards' broken out to a seperate component.